web/src/main/java/it/pagopa/selfcare/party/registry_proxy/web/controller/AOOController.java

```diff
@@ -58,10 +60,17 @@ public AOOsResource findAll(@ApiParam(value = "${swagger.model.*.page}")
     @ApiOperation(value = "${swagger.api.aoo.findBy.summary}",
             notes = "${swagger.api.aoo.findBy.notes}")
     public AOOResource findByUnicode(@ApiParam("${swagger.model.aoo.codiceUniAoo}")
-                                       @PathVariable("codiceUniAoo") String codiceUniAoo) {
+                                     @PathVariable("codiceUniAoo") String codiceUniAoo,
+                                     @ApiParam(value = "${swagger.model.*.categories}")
+                                     @RequestParam(value = "categories", required = false)
+                                     String categories) {
         log.trace("find AOO start");
         log.debug("find AOO codiceUniAoo = {}", codiceUniAoo);
-        final AOOResource aooResource = AOOMapper.toResource(aooService.findByUnicode(codiceUniAoo));
+        List<String> categoriesList = Collections.emptyList();
+        if (StringUtils.hasText(categories)) {
+            categoriesList = Arrays.stream(categories.split(",")).collect(Collectors.toList());
+        }
+        final AOOResource aooResource = AOOMapper.toResource(aooService.findByUnicode(codiceUniAoo,categoriesList));
```

Comment:
look at https://medium.com/@AADota/spring-passing-list-and-array-of-values-as-url-parameters-1ed9bbdf0cb2